### ISSUE TITLE
Exception while printing help message

### ISSUE THREAD

**Issue by konsumlamm** [Kotlin/dokka#666]
**Describe the bug**
An exception occurs when trying to print the help message using the Dokka jar.
```
$ java -jar dokka-fatjar-0.10.1.jar --help
Usage: dokka [-h] [-output] [-format] [-generateIndexPages] [-cacheRoot] [-pass] [-globalPackageOptions] [-globalLinks] [-globalSrcLink] 

  -h, --help                Prints help
  -output                   Output directory path
  -format                   Output format (text, html, markdown, jekyll, kotlin-website)
  -generateIndexPages       Generate index page
  -cacheRoot                Path to cache folder, or 'default' to use ~/.cache/dokka, if not provided caching is disabled
  -pass                     Single dokka pass
  -globalPackageOptions     List of package passConfiguration in format "prefix,-deprecated,-privateApi,+warnUndocumented,+suppress;..." 
  -globalLinks              External documentation links in format url^packageListUrl^^url2...
  -globalSrcLink            Mapping between a source directory and a Web site for browsing the code
Exception in thread "main" kotlinx.cli.HelpPrintedException
	at kotlinx.cli.FlagArgumentsKt$help$1.invoke(FlagArguments.kt:100)
	at kotlinx.cli.FlagAction$DefaultImpls.invoke(Actions.kt:9)
	at kotlinx.cli.FlagActionBase.invoke(FlagArguments.kt:4)
	at kotlinx.cli.CommandLineParser.doParse(CommandLineParser.kt:56)
	at kotlinx.cli.CommandLineParser.parseTokenized(CommandLineParser.kt:21)
	at kotlinx.cli.CommandLineParser.parse(CommandLineParser.kt:11)
	at kotlinx.cli.CommandLineParser.parse(CommandLineParser.kt:7)
	at kotlinx.cli.ParseKt.parseArgs(parse.kt:4)
	at org.jetbrains.dokka.ParseContext.parse(DokkaArgumentsParser.kt:54)
	at org.jetbrains.dokka.DokkaArgumentsParser.parseInto(DokkaArgumentsParser.kt:118)
	at org.jetbrains.dokka.MainKt.createConfiguration(main.kt:261)
	at org.jetbrains.dokka.MainKt.main(main.kt:269)
```

**Expected behaviour**
No exception occurs.

**Installation**
- Operating system: Linux
- Dokka version: 0.10.1

**Comment by michelou** [comment]
Same issue on Windows 10 (Dokka 0.10.1).

Stack trace is slightly different:
```java
Exception in thread "main" kotlinx.cli.HelpPrintedException
        at kotlinx.cli.CommandLineParser.parseTokenized(CommandLineParser.kt:17)
        at kotlinx.cli.CommandLineParser.parse(CommandLineParser.kt:11)
        at kotlinx.cli.CommandLineParser.parse(CommandLineParser.kt:7)
        at kotlinx.cli.ParseKt.parseArgs(parse.kt:4)
        at org.jetbrains.dokka.ParseContext.parse(DokkaArgumentsParser.kt:54)
        at org.jetbrains.dokka.DokkaArgumentsParser.parseInto(DokkaArgumentsParser.kt:118)
        at org.jetbrains.dokka.MainKt.createConfiguration(main.kt:261)
        at org.jetbrains.dokka.MainKt.main(main.kt:269)
```

**Comment by kamildoleglo** [comment]
Fixed in `1.4.10.2`
